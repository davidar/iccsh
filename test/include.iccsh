:include <stdlib.h>
:printf %p NULL

:include math
:printf %.16f M_PI

:include "test.h"
:printf %d LIFE
