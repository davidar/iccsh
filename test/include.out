(nil)
3.1415926535897931
42
